server/db/scripts/seed.js

```diff
@@ -2,50 +2,32 @@
 const { dbClient } = require('../db.js');
 const { schema } = require('../schema.js');
 
+//Postgres duplicated error code
```

Comment:
Couple of tiny thoughts for this file, I'm cool if we don't want to address these:
- there's a missing space after the // on this line
- since we've had some on-and-off issues with local dev, I think it might be nice to put some logging after each successful operation in this script (e.g. table created, index created)

Reply:
No problem. Just did :)

Reply:
Looks great! :)